Sources/Camille/main.swift

```diff
@@ -6,20 +6,25 @@ let bot = try SlackBot(
     authenticator: TokenAuthentication.self,
     storage: RedisStorage.self,
     services: [
-        AnnouncementService(config: AnnouncerConfig { im in
-            return SlackMessage(target: im)
-                .text("Hi,")
-                .user(im.user)
-                .text(", welcome to the ios-developer slack team!")
-                .apiMethod()
-        }),
         HelloBot(),
         KarmaBot(options: KarmaBotOptions(
             targets: ["bot-laboratory"],
             addText: "++",
             removeText: "--",
             textDistanceThreshold: 4
-        ))
+        )),
+        TimedMessageService(config: TimedMessageConfig(interval: 3600.0, target: "intro") { target in
```

Comment:
Drop this for now, the admins are drafting an announcement which will be added later